non-msrv-tests/tests/ui/item.stderr

```diff
@@ -14,38 +14,68 @@ error: empty `derive_where` found
   |
   = note: this error originates in the attribute macro `derive_where` (in Nightly builds, run with -Z macro-backtrace for more info)
 
+error: the `crate` option has to be defined in it's own `#[derive_where(..)` attribute
+  --> tests/ui/item.rs:11:16
+   |
+11 | #[derive_where(crate(derive_where_))]
+   |                ^^^^^
```

Comment:
This looks a bit strange, shouldn't the error be:

> crate has do be defined as `crate="smth"` and not `crate(smth)`?

Because it is in its own derive_where attribute

Reply:
Yes, I forgot to mention, basically I was sick of creating new code branches to handle errors. The code is becoming really bloated because of it. I will look into it again.